### DISCUSSION TITLE
Host key verification fail

### DISCUSSION THREAD

**Discussion by Dreadox** [planetarium/libplanet#1863] [Show and tell]
Hello guys, im trying to "compile" this game, but when im doing git submodule update --init --recursive , it gives me Host verification Fail. Did someone pass this ? Can someone tell me how can i do it please? Thank you in advance

**Comment by dahlia** [comment]
I guess you're talking about *Nine Chronicles* rather than *Libplanet*.  We're going to transfer it to the appropriate repository.
